minigun/minigun/cuda/advance_all.cuh

```diff
@@ -116,11 +193,24 @@ void CudaAdvanceAll(
         << " but only got a buffer of length " << output_frontier->length;
     }
   }
-  IntArray1D<Idx> outbuf = (output_frontier)? *output_frontier : IntArray1D<Idx>();
+   */
+  IntArray1D<Idx> outbuf = IntArray1D<Idx>(); //(output_frontier)? *output_frontier : IntArray1D<Idx>();
```

Comment:
change back?

Reply:
removed `output_frontier` in AdvanceAll.